### DISCUSSION TITLE
Any update on flexdashboard for quarto?

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#3271] [Q&A] [answered]
Any update on flexdashboard for quarto?

**Comment by mcanouil** [comment]
This feature as well as Paged.js will be very likely advertised by the team when they'll release them.
Be patient! 😉

**Comment by ManuelSpinola** [comment]
Thank you very much Mickaël.



On Thu, 10 Nov 2022 at 14:23 Mickaël Canouil ***@***.***>
wrote:

> This features as well as Paged.js will be very likely advertise by the
> team when they'll release them.
> Be patient! 😉
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/3271#discussioncomment-4111174>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FB7MELGRYBOPIAPZFOLWHVKTJANCNFSM6AAAAAAR4VL2MM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app

**Comment by PeneLoopy** [comment]
I see that this question was asked in November and I'm wondering if the Quarto team has additional information.  I use flexdashboard in my course and cannot switch from R Markdown to Quarto until this option is available.  I'm hoping it will be available by May or June of 2023 so that I have time to revise all of my course material for the fall semester.  Thank you.

**Reply by cscheid** [reply]
Thanks for the interest in quarto! We'll say more when we know more. What we can share right now is that we do have a plan for a dashboarding feature in quarto, and we have a workable design. We are currently implementing the necessary infrastructure to be able to support that design, and that will take a while.

I will say, though, that the dashboarding feature in quarto won't be exactly a reimplementation of flexdashboard. So if you need full backwards compatibility, you're likely better off with RMarkdown for the near-term future.

**Reply by PeneLoopy** [reply]
Thank you.  That's really helpful.  I plan to switch my slides for all of my courses to Quarto this summer but I'll wait on teaching the students Quarto until the new dashboarding feature is available and fully documented.  Thank you, again.

**Comment by marcdotson** [comment]
Any chance we'll be hearing about dashboarding with Quarto by posit::conf(2023)?

**Comment by jhchou** [comment]
Today is my one-year anniversary of starting to use Quarto (when I migrated my low activity blog from Hugo / Blogdown to Quarto).

I was, and remain, impressed and super hopeful about Quarto, but pretty much the _**one**_ thing preventing me from making a full migration from `.rmd` to `.qmd` is the lack of a replacement for `flexdashboard`.

Hoping for an update soon!

**Comment by nicholasjhorton** [comment]
I too would welcome any updates: this is also one of the last Rmd remnants that I still have.

**Comment by jjallaire** [comment, marked as answer]
Quarto Dashboards are available in the v1.4 pre-release: https://quarto.org/docs/dashboards/

**Reply by jhchou** [reply]
Thank you so much!!! Looking forward to trying the pre-release.

Has anyone summarized major functional differences / capabilities between Rmd `flexdashboard` and Quarto `dashboard`?

**Reply by marcozanotti** [reply]
It would be very much appreciated some documentation related to pros and cons comparing the two, flexdashboard and quarto dashboard. 
By the moment it is difficult to understand whether quarto dashboard can do everything flexdashboard is doing. There are some amazing features of quarto dashboard that are very appealing, hence it would be helpful to understand if one can safely switch to it.

**Comment by ManuelSpinola** [comment]
Thank you very much!
